backend/runprocessors.py

```diff
@@ -99,6 +126,9 @@ def run_manual_fixes():
         "program": cache_program_mappings,
         "graph": cache_graph
     },
+    "enrolment": {
+        "scrape": run_scrape_enrolment_data,
```

Comment:
Have a more explicit error here instructing the user to add username and password if not given.